js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -672,6 +686,8 @@ const TreeViewBase = HierarchicalCollectionWidget.inherit({
 
         $node.addClass(showCheckBox ? ITEM_WITH_CHECKBOX_CLASS : ITEM_WITHOUT_CHECKBOX_CLASS);
         $node.toggleClass(INVISIBLE_STATE_CLASS, nodeData.item.visible === false);
+
+        this._showCustomExpanderIcons() && $node.addClass(ITEM_WITH_CUSTOM_EXPANDER_ICON_CLASS);
```

Comment:
It's not a good practice to use logical operator instead of usual statement
https://github.com/airbnb/javascript#control-statements--value-selection
```suggestion
        if(this._showCustomExpanderIcons()) {
            $node.addClass(ITEM_WITH_CUSTOM_EXPANDER_ICON_CLASS);
        }
```